test/test_p2p.py

```diff
@@ -2,18 +2,57 @@
 import sys
 import datetime
 import pytest
-from p2p import get_latest_report_date
+from p2p import get_latest_report_date, filter_investment_files_by_date
+
+INPUT_DATA = {
```

Comment:
It is ok to extract this data but now this is not input data anymore in all cases. For example in test_filter_investment_files_by_date_newest the input data is platform_data.
IMHO it is better for the tests to follow a patter regardin input and output data. For example input_data and expected_output, with this way is easier faster to follow and read when you have a lot of tests and you do not have ot think if the platform_data is the input or the output.